tests/libvmi/vmi.go

```diff
@@ -286,6 +286,12 @@ func WithEvictionStrategy(evictionStrategy v1.EvictionStrategy) Option {
 	}
 }
 
+func WithStartStrategy(startStrategy v1.StartStrategy) Option {
```

Comment:
This contradicts rule 2 of https://github.com/kubevirt/kubevirt/tree/main/tests/libvmi#rules

As this is a small one and indeed hard to locate, I vote to relocate in a follow up and let this in as is for now.